spikes/api/main.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	mm "github.com/mattermost/platform/model"
+)
+
+/*
+main
+Usage: go run main.go -u <username> -p <password> <server-url> [team-name]
+Authenticates your login information, then gives you your AuthToken.
+If the team name is unentered or invalid main shows valid team names.
+*/
+func main() {
+	username := flag.String("u", "", "Username")
+	password := flag.String("p", "", "Password")
+	flag.Parse()
+	url := flag.Arg(0)
+	teamName := flag.Arg(1)
+	client := mm.NewClient(url)
+	_, err := client.Login(*username, *password)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	fmt.Println("Auth successful! Token: ", client.AuthToken)
```

Comment:
Strictly speaking, we probably don't need to print this anymore. It was there originally to prove that we had logged in successfully, but now (thanks to @aThorp96) we have much better ways to prove that.